playground/html/__tests__/html.spec.ts

```diff
@@ -326,11 +326,29 @@ describe('importmap', () => {
   })
 })
 
+describe.runIf(isServe)('url with pathname', () => {
+  test('pathname conflicts with folder at same level as base, should load index.html', async () => {
+    await page.goto(viteTestUrl + '/side-effects')
+    expect(browserLogs).toContain('pathname:/side-effects')
+  })
+  test('pathname conflicts with folder at same level as base, should load index.html', async () => {
```

Comment:
This description seems incorrect?

Reply:
It has been updated with a more precise description of this test case.